pom.xml

```diff
@@ -55,6 +56,17 @@
   <dependencies>
   </dependencies>
 
+  <build>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <artifactId>maven-compiler-plugin</artifactId>
+          <version>3.13.0</version>
```

Comment:
Extract a property instead 😉

Reply:
actually I will need to properly update this to a new vertx parent, this is still a draft PR kind off, as I want to prepare the vertx-rx / vertx-kotlin for code migration